src/rgw/rgw_lua_request.cc

```diff
@@ -757,6 +757,8 @@ struct RequestMetaTable : public EmptyMetaTable {
       pushstring(L, s->dialect);
     } else if (strcasecmp(index, "Id") == 0) {
       pushstring(L, s->req_id);
+    } else if (strcasecmp(index, "UserId") == 0) {
+      pushstring(L, s->user->get_id().id);
```

Comment:
can we use the `UserMetaTable` instead?
we will have `Request.User` here, and then `User.Id` and `User.Tenant` similar to other places where we have "User"